packages/dashboard/src/app/views/editorSettings/archive/index.js

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import { useCallback } from '@web-stories-wp/react';
+import { __, _x, TranslateWithMarkup, sprintf } from '@web-stories-wp/i18n';
+import { useFeature } from 'flagged';
+import { DropDown, THEME_CONSTANTS } from '@web-stories-wp/design-system';
+
+/**
+ * Internal dependencies
+ */
+
+import {
+  InlineLink,
+  SettingForm,
+  SettingHeading,
+  TextInputHelperText,
+} from '../components';
+import { ARCHIVE_TYPE } from '../../../../constants';
+
+export const TEXT = {
+  LABEL: __('Stories Archives', 'web-stories'),
+  SECTION_HEADING: __('Stories Archives', 'web-stories'),
+  ARCHIVE_CONTENT: __(
+    "By default WordPress automatically creates an archive page, displaying your latest stories in your theme's layout.",
+    'web-stories'
+  ),
+  SUB_TEXT_DISABLED: __('Disabled the default archive page.', 'web-stories'),
```

Comment:
After testing, let's change this text a bit:

```suggestion
  SUB_TEXT_DISABLED: __('This disables the default archive page.', 'web-stories'),
```